### ISSUE TITLE
Prices start getting a lot of decimals/zeros when switch between auctions

### ISSUE THREAD

**Issue by elena-zh** [gnosis/ido-ux#345]
1. Open auction 1
2. then changing an URL, open an auction 2

**AR**: start getting a lot of decimals/zeros when switch between auctions
https://drive.google.com/file/d/1QMsp868ST13_n9I0vX9Wi3FWZSBbEAFm/view

**ER**: min prices show correct values

**Comment by gabitoesmiapodo** [comment]
This no longer seems to be an issue.

@elena-zh please have a look when you have the chance.

**Comment by elena-zh** [comment]
@gabitoesmiapodo , 
I'm able to reproduce the issue: 
![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/fc8b2b95-eb74-40a2-b240-620ac666f782)
Please, follow the exact steps in order to reproduce it and change an auction with changing auditonID on the URL.

**Comment by gabitoesmiapodo** [comment]
@elena-zh 

This is what I see (this is on my local machine, though, currently on the `/develop` branch):

https://user-images.githubusercontent.com/4015436/114192640-1b404000-9924-11eb-9945-bc0411312338.mp4

Is this the right procedure or am I doing something different? If this is OK, it could be that the development server is not updated, but I'm not sure.

**Comment by elena-zh** [comment]
Yes, the behavior is correct on your local machine. Hence, dev env-t is not yet updated due to I'm still able to reproduce the issue there.

**Comment by elena-zh** [comment]
Now prices are changing, however, a lot of zeros/decimals appear in them. 
I have updated the issue description.
